services/web-server/src/loaders/workerManager.js

```diff
@@ -12,6 +12,16 @@ module.exports = ({ workerManager }) => {
   //   )
   // );
 
+  // const WMWorkers = new DataLoader(queries =>
+  //   Promise.all(
+  //     queries.map(async (workerType, isQuarantined, filter) => {
+  //       const workers = await workerManager.getWorkers(provisionerId, workerType, isQuarantined);
+  //
+  //       return filter ? sift(filter, workers) : workers;
+  //     })
+  //   )
+  // );
```

Comment:
Do you want me to look at the validity of these comments?

Reply:
not yet - these are just sketches at the moment. I plan to get them finished when I switch to API, and when I'll clean up this from the mock data